pipeline.ipynb

```diff
@@ -2,606 +2,618 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "id": "ad9a2efe-9ebc-4234-b418-81db4726fe1c",
+   "id": "0e00cc99",
    "metadata": {
     "tags": []
    },
    "source": [
     "# AK spruce beetle outbreak risk pipeline\n",
     "\n",
-    "This notebook is used for creating a climate-based dataset of spruce beetle outbreak risk.\n",
+    "This notebook constitutes the pipeline for producing a dataset of projected climate-driven risk of spruce beetle outbreak for forested areas of Alaska for the 21st century. See the [README](README.md) for more information.\n",
     "\n",
-    "It is currently under development."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 19,
-   "id": "d842643b-aaa1-4b90-8fd3-ba03cc94ff00",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "import numpy as np\n",
-    "import xarray as xr\n",
-    "from pathlib import Path"
+    "### Outputs\n",
+    "\n",
+    "The main product of this pipeline is a 5-D datacube of one categorical variable - climate-driven spruce beetle outbreak risk. The dimensions are:  \n",
```

Comment:
Looks like the datacube has 6 axes. Should this say "6-D datacube"?

Reply:
yes!